internal/runners/push/push.go

```diff
@@ -232,6 +239,29 @@ func (r *Push) getNamespace() (*project.Namespaced, error) {
 	return ns, nil
 }
 
+func (r *Push) promptNamespace() (*project.Namespaced, error) {
+	owner := authentication.Get().WhoAmI()
```

Comment:
Since you're calling this from inside the forking flow the namespace param seems to be ignored. Meaning that even when someone run `state push foo/bar` it will still prompt for an owner/name.

Reply:
If a namespace parameter is passed then it is likely that the user will have permission to create a project under the given org with the given name. In that case, we will not hit the fork logic at all.

If the user does not have permission to create a project with the namespace they passed in then we will have to prompt them for a new one. So in this case I think we would _want_ to ignore their parameters. Or we could at least use the name as the default in the prompt as they are not authorized to create a project with the given org.